contrib/internet-urllib3.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+import urllib3
+import certifi
+
+def isInternetConnected(url="www.ietf.org"):
+  result = False
+  http = urllib3.PoolManager(
+    cert_reqs='CERT_REQUIRED', # Force certificate check.
+    ca_certs=certifi.where(),  # Path to the Certifi bundle.
+  )
+  try:
+    r = http.request('HEAD', 'https://' + url)
+    result = True
+  except Exception as e:  # urllib3.exceptions.SSLError
+    result = False
```

Comment:
The issue is about false negatives. Users will hit this issue and we will have complaints about that.

Anyway, this is not critical. It's possible to implement other strategies and let the users configure what's best for them.